ghascompliance/__main__.py

```diff
@@ -198,6 +205,10 @@
 
     Octokit.info("Total unacceptable alerts :: " + str(errors))
 
+    # Summary and PR comment
+    Summary.outputJobSummary()
+    PullRequest.addPrComment(policy.name)
```

Comment:
It might be interesting to see what happens / what we can do if an error is hit in the checks (exception was thrown). Do we comment something in the PR in this case?

Reply:
Good point, if an exception is raised before this point then nothing will be output to the job summary and PR comment.
Could add a try catch around the checks above this point. Then either add some info to the summary or just output what is there before re-raising the exception.

Reply:
Maybe something like this?
![image](https://github.com/advanced-security/policy-as-code/assets/25668381/20cf5596-0d91-467f-b67a-528a9cac23df)

Reply:
Maybe in this except block?

https://github.com/advanced-security/policy-as-code/blob/0c3c71ffebf519b067de0060e6d39a987e30a1e7/ghascompliance/__main__.py#L199-L201

Reply:
I think for now we'll leave it, amazing work!